frontends/precinct-scanner/src/screens/scan_warning_screen.test.tsx

```diff
@@ -59,9 +61,9 @@ test('blank ballot', () => {
     adjudicationReasonInfo: [{ type: AdjudicationReason.BlankBallot }],
   });
 
-  screen.getByRole('heading', { name: 'Blank Ballot' });
-  screen.getByText('Your ballot does not have any votes.');
```

Comment:
I think "Your ballot does not have any votes" is plainer language. It uses active voice and also the possessive "your ballot", both of which are recommended in the plain language guidelines. ([use of pronouns](https://www.plainlanguage.gov/guidelines/audience/address-the-user/)) ([active voice](https://www.plainlanguage.gov/guidelines/conversational/use-active-voice/))